### ISSUE TITLE
Choose some grid filter in the user profile : get 500

### ISSUE THREAD

**Issue by naolis** [akeneo/pim-community-dev#4994]
Hi, 

In Akeneo CE 1.6.2

Go to "My account", "edit", go to "Additional", add some "Product grid filters (required)"
In the icecat demo context i try with : statut, sku, description, family

When you "Save" get an error 500

After that if i try to go to my account in app_dev context i get the following 
`Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("You have requested a non-existent service "pim_datagrid.datagrid.product.filters_configurator". Did you mean this: "pim_datagrid.datagrid.configuration.product.filters_configurator"?") in OroUserBundle:User:view.html.twig at line 110.`

The URL is 
/app_dev.php#url=/app_dev.php/user/profile/view

**Comment by nidup** [comment]
Hi @naolis 

Thank you for raising this issue! We well reproduced the issue on the v1.6 release :tada: 

Are you interested to investigate and contribute to fix it by opening a pull request on the 1.6 branch? 

I'll be happy to help you by reviewing and merging your pull request. To be able to merge it I need that you read and sign our contributor agreement https://www.akeneo.com/contributor-license-agreement/.

Best regards

**Comment by nidup** [comment]
In fact, it seems that we use a deprecated service, the following change allows to fix the issue,

```
diff --git a/src/Pim/Bundle/DataGridBundle/Twig/FilterExtension.php b/src/Pim/Bundle/DataGridBundle/Twig/FilterExtension.php
index f49107f..d8ecb3c 100644
--- a/src/Pim/Bundle/DataGridBundle/Twig/FilterExtension.php
+++ b/src/Pim/Bundle/DataGridBundle/Twig/FilterExtension.php
@@ -75,6 +75,6 @@ class FilterExtension extends Twig_Extension
      */
     final protected function getFiltersConfigurator()
     {
-        return $this->container->get('pim_datagrid.datagrid.product.filters_configurator');
+        return $this->container->get('pim_datagrid.datagrid.configuration.product.filters_configurator');
     }
 }
diff --git a/src/Pim/Bundle/DataGridBundle/spec/Twig/FilterExtensionSpec.php b/src/Pim/Bundle/DataGridBundle/spec/Twig/FilterExtensionSpec.php
index 4b4e3e9..8c95fe5 100644
--- a/src/Pim/Bundle/DataGridBundle/spec/Twig/FilterExtensionSpec.php
+++ b/src/Pim/Bundle/DataGridBundle/spec/Twig/FilterExtensionSpec.php
@@ -19,7 +19,7 @@ class FilterExtensionSpec extends ObjectBehavior
         Manager $manager
     ) {
         $container->get('oro_datagrid.datagrid.manager')->willReturn($manager);
-        $container->get('pim_datagrid.datagrid.product.filters_configurator')->willReturn($configurator);
+        $container->get('pim_datagrid.datagrid.configuration.product.filters_configurator')->willReturn($configurator);
         $this->beConstructedWith($container);
     }
```

**Comment by nidup** [comment]
Hi, the issue has been fixed by @julienanquetil and the fix will be available in the upcoming 1.6.3 :tada: :tada: :tada:
